setup.cfg

```diff
@@ -1,14 +1,16 @@
 [metadata]
 name = matrix_common
-description = "Common utilities for Synapse, Sydent and Sygnal"
+description = Common utilities for Synapse, Sydent and Sygnal
 long_description = file: README.md
 long_description_content_type = text/markdown
-version = 0.0.0
+version = 1.1.0
```

Comment:
Should this be just 1.0?

Reply:
I've been silly and published a bad version of `1.0.0`

I went with a 3-component version based off @DMRobertson's comment here:
https://github.com/matrix-org/synapse-s3-storage-provider/blob/main/setup.py#L3

Reply:
I was question the 1.1-ness, not the number of components. I'm guessing we can reset this when switching to real PyPI instead of test PyPI?

Reply:
To clarify, I've been *very* silly and published a bad version of 1.0.0 to the real PyPI.

Reply:
This seems to be at https://pypi.org/project/matrix-python-common/? But if the name is now `matrix-common` that doesn't matter... I think you can also delete projects somewhere in PyPI.

Reply:
Ooh you're right. I've gotten myself confused.